sigs/community-outreach.md

```diff
@@ -0,0 +1,17 @@
+# Konflux Community Outreach Special Interest Group
+
+## Purpose
+
+Foster a self-sustaining open source community for Konflux composed of its contributors, end users,
+evaluators, and adopters. This SIG aims to accomplish this mission as follows:
+
+* Encourage open source best practices within the Konflux project and its sub-projects/dependencies.
+* Establish a communication strategy for Konflux knowledge sharing.
+* Promote Konflux innovations through public speaking opportunities, blog posts, and other media.
+* Identify opportunities for collaboration with other projects, initiatives, and institutions.
+
+## Communication
+
+* Email the [Konflux mailing list](https://groups.google.com/g/konflux), optionally adding
+  `[sig-community-outreach]` to the subject header.
+* We will host a recurring open SIG meeting at a time TBD.
```

Comment:
Do you want to add a note here that this file will be updated once that time is determined?

Reply:
I think that is implied, but I can make that update explicit if folks think it is necessary.

Reply:
Let's go for it and merge this. We'll follow up soon with a time.